boards/arm/xilinx_kv260_r5/doc/index.rst

```diff
@@ -0,0 +1,99 @@
+.. _xilinx_kv260_starter_kit_r5:
+
+Xilinx KV260 Development Board RPU Cortex-R5
+############################################
+
+Overview
+********
+This configuration provides support for the RPU, real-time processing unit on Xilinx
+KV260 devices, it can operate as following:
+
+* Two independent R5 cores with their own TCMs (tightly coupled memories)
+* Or as a single dual lock step unit with double the TCM size.
+
+This processing unit is based on an ARM Cortex-R5 CPU, it also enables the following devices:
+
+* ARM PL-390 Generic Interrupt Controller
+* Xilinx Zynq TTC (Cadence TTC)
+* Xilinx Zynq UART
+
+Hardware
+********
+Supported Features
+==================
+
+The following hardware features are supported:
+
++--------------+------------+----------------------+
+| Interface    | Controller | Driver/Component     |
++==============+============+======================+
+| GIC          | on-chip    | generic interrupt    |
+|              |            | controller           |
++--------------+------------+----------------------+
+| TTC          | on-chip    | system timer         |
++--------------+------------+----------------------+
+| UART         | on-chip    | serial port          |
++--------------+------------+----------------------+
+
+The kernel currently does not support other hardware features on this platform.
+
+Devices
+========
+System Timer
+------------
+
+This board configuration uses a system timer tick frequency of 1000 Hz.
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+on-chip UART1.
+
+Known Problems or Limitations
+==============================
+
+The following platform features are unsupported:
+
+* Dual-redundant Core Lock-step (DCLS) execution is not supported yet.
+* Xilinx Zynq TTC driver does not support tickless mode operation.
+
+Programming and Debugging
+*************************
+Currently the best way to run this sample is by loading it through remoteproc
+from APU which runs Linux to the RPU, so assuming the target board has a compatible
+Linux kernel (Versions 5.15 from Xilinx abd 6.2 are supported), first build the app.
```

Comment:
Can you add a link to somewhere users can find instructions on downloading/building and flashing the SD card (etc.) with an appropriate Linux image, to at least point them in the right direction?